packages/amplify-provider-awscloudformation/src/network/stack.ts

```diff
@@ -0,0 +1,196 @@
+import * as apigw2 from '@aws-cdk/aws-apigatewayv2';
```

Comment:
Ideally we put in all the cfn resource generation logic pertaining to a category into a category plugin itself. Any specific reason why we're doing it all in the amplify-provider-awscloudformation plugin instead?

Reply:
The idea is that the network stack can be reused by other categories.

On the other hand, containers specific stuff goes into api category (EcsStack)